### PR TITLE
KIN-76 navigation bar public view front end

### PR DESCRIPTION
**Pull request by CarlosMay7**
[KIN-75](https://loopcrack.atlassian.net/browse/KIN-75) Large [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

The project needs a way to reach the different pages of the site

### Solution

Create a navigation bar that contains the names of the different pages of the site and redirects the user to each one.

### Requirements checklist

- [x]  It must be displayed on each public page of the website.

- [x] It must include the company logo, which when clicked on it redirects to the main page of the site.

- [x]  The navigation bar should include labeled links to access the following public pages: Equipment, Technical Support, Certificates, and Contact.

- [x]  The "Contact" link should redirect the user to the home page and be automatically positioned in the contact form.

- [x]  The appearance of the navigation bar must be responsive.

### Testing

- [x] Tested the redirection of the logo and the contact link

## Demo

https://github.com/loopcrack-org/Kinub/assets/97546829/212438d3-f329-47bb-8804-3b0c80b20715

image

image

[KIN-75]: https://loopcrack.atlassian.net/browse/KIN-75?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by CarlosMay7** [comment]
The logo is unchecked because the question arose if we have to make the logo according to the mockups or they will send us an image of it

**Review by Mauriciocr207 — APPROVED** [review]

**Review by Oswaldo-Chan — APPROVED** [review]

**Review by LUGAMAFE — CHANGES_REQUESTED** [review]
@CarlosMay7 can we reduce the height of the header to only 60px instead of 80px, other thing that is missing is that the header should be fixed and follow you all along the page, also for the moment it is ok to omit the logo, or put any other logo as an example ;)

**Review by FernandoJoachin — COMMENTED** [review]
Encontré una interacción rara con el botón de la navegación. Lo que sucede es que si tengo la navegación en formato de celular, luego le pico al botón para desplegar la navegación, después redimensiono la ventana a un formato de tablet o laptop, y regreso al formato de celular, el botón hace la acción contraria, es decir, la X despliega la navegación y las 3 rayitas quitan la navegación.

**Review by Cristianpan — APPROVED** [review]

**Review by CarlosJCD — APPROVED** [review]

File: src/sass/public/UI/header/_sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    &.show {
+        padding: 10rem 2rem 2rem 2rem;
+        opacity: 1;
+        height: 100%;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: rgb(0 0 0 / 0.90);
+        z-index: 2;
```
Comment (LUGAMAFE) [inline src/sass/public/UI/header/_sidebar.scss:19]: Could you add please:
```suggestion
        z-index: 2;
        backdrop-filter: blur(3px);
```

File: src/sass/public/UI/header/_header-mobile.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: sticky;
+    width: 100%;
+    height: 6rem;
+    top: 0;
+    z-index: 100;
+    background-color: #FFF;
+    box-shadow: 0px 10px 15px 0px v.$gray;
```
Comment (LUGAMAFE) [inline src/sass/public/UI/header/_header-mobile.scss:15]:
```suggestion
```
remove this line

File: src/sass/public/UI/header/_sidebar.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    &.show {
+        padding: 10rem 2rem 2rem 2rem;
+        opacity: 1;
+        height: 100%;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: rgb(0 0 0 / 0.90);
```
Comment (LUGAMAFE) [inline src/sass/public/UI/header/_sidebar.scss:18]:
```suggestion
        padding: 2rem; // Dont use excesive padding
        opacity: 1;
        height: 100%;
        width: 100%;
        position: fixed;
        top: 60px; // create a sass or css variable to save the header height
```
dont use excesive padding use the header size instead. Create a sass or css variable to save the header height for mobile and desktop

File: src/sass/public/UI/header/_header.scss [hunk 4]
```diff
@@ -0,0 +1,176 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.header {
+    box-shadow: 0px 10px 15px 0px v.$gray;
```
Comment (LUGAMAFE) [inline src/sass/public/UI/header/_header.scss:5]:
```suggestion
    box-shadow: 0px 6px 15px 0px v.$gray-500;   
```
reduce the box shadow

**Review by LUGAMAFE — CHANGES_REQUESTED** [review]
@CarlosMay7 @Oswaldo-Chan the instruction was that on desktop still have a height of 80px and on mobile and tablet 60px. likewise in galaxy fold resposive design is incorrect. If I'm honest the white shadow that comes out of the mobile view seems ugly and unnecessary. I think we can lower the intensity of the navbar shadow in the desktop view. Other than that good job.

**Review by LUGAMAFE — APPROVED** [review]
@CarlosMay7 Listo muchas gracias por los cambios nadamas corrigue el nombre del PR antes de mergear porfavor

**Review by CheloChidoone — APPROVED** [review]

**Pull request merged**